site/components/Navigation/NavItemMobile.tsx

```diff
@@ -1,26 +1,39 @@
-import Link from "next/link";
-import React, { FC } from "react";
+import React, { Dispatch, FC, SetStateAction, ReactElement } from "react";
 import * as styles from "./styles";
-
+import { cx } from "@emotion/css";
+import { ButtonPrimary } from "@klimadao/lib/components";
+import { NavItem } from "./index";
 interface Props extends React.AnchorHTMLAttributes<HTMLAnchorElement> {
   name: string;
+  id: NavItem;
   url: string;
-  number?: string;
+  selected: string | undefined;
+  setSelected: Dispatch<SetStateAction<NavItem | undefined>>;
```

Comment:
setSelected [does this](https://github.com/KlimaDAO/klimadao/pull/591/files#diff-75ddc86bebead12e067cb6c4b85471f1e38efb0c48f38e6f4ef3f8a69fc85e2cR26) and does not know about any state update in a parent component.


So the correct type would be  `setSelected: (string: NavItem) => void;`

Reply:
type updated, ty for the explanation